src/antsibull_fileutils/copier.py

```diff
@@ -95,7 +95,7 @@ def copy(self, from_path: StrPath, to_path: StrPath) -> None:
 
             # Copy the file
             dst_path = os.path.join(to_path, file_decoded)
-            shutil.copyfile(src_path, dst_path)
+            shutil.copyfile(src_path, dst_path, follow_symlinks=False)
```

Comment:
What if it's a relative symlink that points to outside the git checkout? Is this something we want to support?

Also, I'd replace `copyfile` with `copy2` that preserves file metadata (probably in a separate PR).

Reply:
> What if it's a relative symlink that points to outside the git checkout? Is this something we want to support?

If it points outside the git checkout then IMO it can be expected to be broken. If the collection is in a subdirectory of a git repository and the symlink points to a file in the repo outside the collection, well, that's a tricky case. I think for now it's OK to break it.

> Also, I'd replace `copyfile` with `copy2` that preserves file metadata (probably in a separate PR).

:+1: